task/build-image/0.1/README.md

```diff
@@ -0,0 +1,61 @@
+# Image Builder
+
+The image-builder Task builds an OCI image from a Dockerfile.
```

Comment:
There's inconsistency in naming convention for image-builder. Additionally I think this should be stated that this is a task in the title

Reply:
This is a task documentation. No need to repeat that in the title. Aligned with other docs